src/main/java/com/sprint/mission/discodeit/service/file/FileChannelService.java

```diff
@@ -0,0 +1,103 @@
+package com.sprint.mission.discodeit.service.file;
+
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.service.ChannelService;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+public class FileChannelService implements ChannelService {
+    private Map<UUID, Channel> data;
+    private FileUserChannelService fileUserChannelService = new FileUserChannelService();
+    private FileMessageService fileMessageService = new FileMessageService();
+
+    // 생성자
+    public FileChannelService() { data = loadData(); }
+
+    // *저장로직* 채널 생성
+    @Override
+    public Channel createChannel(String name, String information){
+        Channel newChannel = new Channel(name, information);
+        data.put(newChannel.getId(), newChannel);
+        saveData();
+        return newChannel;
+    }
+
+    // *저장로직* 전체 채널 조회
+    @Override
+    public List<Channel> findAll(){
+        return data.values().stream().toList(); // 아무 채널도 존재하지 않을 시 빈 list 반환
```

Comment:
[List.copyOf()](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/List.html#copyOf(java.util.Collection))을 사용하면 stream을 매번 돌지 않아도 될 것 같아요.
```suggestion
        return List.copyOf(data.values());
```